index.js

```diff
@@ -625,7 +618,6 @@ Router.prototype.find = function find (method, path, derivedConstraints) {
 
     currentNode = node
 
-    // static route
```

Comment:
why was this removed?

Reply:
I thought it was obvious and didn't think a comment had to be added.
Should I put it back?

Reply:
Please thanks